### PR TITLE
Created appointment model

### PR DESCRIPTION
**Pull request by dapierrot21**
### What this PR does (required):
- Appointment model created

### Screenshots / Videos (required):
- n/a

### Any information needed to test this feature (required):
- n/a

### Any issues with the current functionality (optional):
Did I create the meeting_id correctly?

### CODE REVIEW

File: server/models/Appointment.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        type: String,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true
+    },
+    time: {
+        type: Date,
+        default: Date.now
```
Comment (ajaytomgeorge) [inline server/models/Appointment.js]: Time of the appointment should be _required_. Setting it to default invites erroneous data

Comment (dapierrot21) [reply server/models/Appointment.js]: Very well then. How about the timezone field, should I place a required on it as well? and was the timezoneoffset() used correctly?

File: server/models/Appointment.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const AppointmentSchema = new mongoose.Schema({
+    meeting_id: {
```
Comment (moffatethan) [inline server/models/Appointment.js]: Camel case - `meetingId`

**Review by moffatethan — APPROVED** [review]
LGTM!

**Pull request merged**